docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,130 @@
+# Syncing LDAP Groups to OpenShift Groups
+
+OpenShift cluster admins need a way to sync their OpenShift `Groups` to external records in an LDAP server. This requires that we have a way of retrieving the LDAP users that are the members of a group, determining the OpenShift `Group` name for this collection, and updating the OpenShift record, removing the record as necessary when the corresponding LDAP record is removed. Necessary information about the LDAP server set-up and schema will be provided by the user in a `yaml` configuration file and referenced in the `oadm sync-groups` command.
+
+## Use Cases
+1. I want to sync all LDAP groups minus some blacklist to OpenShift `Groups`
```

Comment:
> not sure * -x -y -z is something we want. all or an explicit list seems more likely.

I don't see a blacklist as unreasonable.  If you have a few, very large, low value groups ("whole company", "by division", "all managers"), you might find it more effective to list the ones you don't want to sync.  It also gives an escape hatch if one particular group is proving troublesome.  Say it makes a `group` too big to fit into etcd as an example.